src/scripts/components/DropdownCheckbox.jsx

```diff
@@ -244,10 +239,10 @@ class DropdownCheckbox extends React.Component {
           {showCheckbox()}
           {hideCaret || (icon && !label && !selectedLabel) ?
             null :
-            <Icon size="small" icon="caret-down" className="dropdown__toggle__caret" />
+            <Icon size="small" icon="caret-down" className="dropdown__toggle__caret dropdown__caret--auto" />
```

Comment:
How does this work with multiple classes on the className attribute? Do these need to be generated with cx method?

Reply:
There was an alignment issue with svg's inside the dropdown button. I fixed this with another css solution and removed the class dropdown__caret--auto from the code. @juanfabrega